### PR TITLE
Divyanshu | Test | Notification icon is not coming on bell icon with new update in change log

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cw02fr4

### CODE REVIEW

File: apps/web-giddh/src/index.stage.html [hunk 1]
```diff
@@ -58,6 +58,7 @@
                 console.error(error);
             }
         </script>
+        <script async src="https://cdn.headwayapp.co/widget.js"></script>
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/index.stage.html]: Do not add new code when we already load the script tag in ngAfterViewInit.

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 2]
```diff
@@ -13893,87 +12440,6 @@ input[type="checkbox"].cdk-visually-hidden::after {
...
-        float: left !important;
-        width: 100% !important;
-        height: 29px !important;
-
-        .mat-form-field-wrapper {
-            height: 100% !important;
-            padding-bottom: 0 !important;
-        }
-
-        .mat-form-field-flex {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/assets/css/style-2.scss:13958]: Please check and do not remove the CSS that is already in use.

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

**Review by dilpreetsinghofficial — APPROVED** [review]

File: apps/web-giddh/src/index.stage.html (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/index.stage.html]: Done

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/assets/css/style-2.scss:13958]: Check these all are unsed responsive code for small devices

**Pull request merged**